inc/saiobject.h

```diff
@@ -249,6 +249,30 @@ sai_status_t sai_query_stats_capability(
         _In_ sai_object_type_t object_type,
         _Inout_ sai_stat_capability_list_t *stats_capability);
 
+/**
+ * @brief Bulk objects get statistics.
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_type Object type
+ * @param[in] object_count Number of objects to get the stats
+ * @param[in] object_key List of object keys
+ * @param[in] number_of_counters Number of counters in the array
+ * @param[in] counter_ids Specifies the array of counter ids
+ * @param[in] mode Statistics mode
+ * @param[out] counters Array of resulting counter values
```

Comment:
Please add that :

- the length of this array should be  object_count * number_of_counters

-  the counter value for ith object's jth statistics is at : 

``` 
 counters[(i*number_of_counters) + j]
```

Reply:
@JaiOCP 
I dont see this comment addressed.

Reply:
Added the comments